scripts/aws/create_cloudformation_stack.py

```diff
@@ -56,21 +57,25 @@ def create_cloudformation_stack(client, stack_name, cft_content, api_token, dc_c
 
 cft['Mappings']['RegionMap'][args.region]['AMI'] = args.ami
 
-egress = cft['Resources']['SecurityGroup']['Properties']['SecurityGroupEgress']
-egress.append(create_egress(args.core_url, 'E2E - Core'))
-egress.append(create_egress(args.optout_url, 'E2E - Optout'))
-egress.append(create_egress(args.localstack_url, 'E2E - Localstack'))
-cft['Resources']['SecurityGroup']['Properties']['SecurityGroupEgress'] = egress
+if args.env == "mock":
+    egress = cft['Resources']['SecurityGroup']['Properties']['SecurityGroupEgress']
+    egress.append(create_egress(args.core_url, 'E2E - Core'))
+    egress.append(create_egress(args.optout_url, 'E2E - Optout'))
+    egress.append(create_egress(args.localstack_url, 'E2E - Localstack'))
+    cft['Resources']['SecurityGroup']['Properties']['SecurityGroupEgress'] = egress
 
 # Now, we overwrite core, optout URL's with bore addresses.
 secrets = cft['Resources']['TokenSecret']['Properties']['SecretString']['Fn::Join'][1]
 core_index = secrets.index('"core_base_url": "')
 secrets = secrets[:core_index] + secrets[core_index+2:]
 optout_index = secrets.index('", "optout_base_url": "')
 secrets = secrets[:optout_index] + secrets[optout_index+2:]
-secrets = secrets[:1] + [f'"core_base_url": "http://{args.core_url}"',f', "optout_base_url":  "http://{args.optout_url}'] + secrets[1:]
+if args.env == "mock":
+    secrets = secrets[:1] + [f'"core_base_url": "http://{args.core_url}"',f', "optout_base_url":  "http://{args.optout_url}'] + secrets[1:]
```

Comment:
Again, can probably use a `protocol` variable - would make it easier to read and see the difference that env makes.